src/foam/nanos/crunch/CapabilityCapabilityJunctionRefine.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'CapabilityCapabilityJunctionRefine',
+  refines: 'foam.nanos.crunch.CapabilityCapabilityJunction',
+
+  documentation: `
+    Refine capabilitycapabilityjunction to add tablecellformatters for source and target id
+  `,
+
+  properties: [
+    {
+      class: 'Reference',
+      of: 'foam.nanos.crunch.Capability',
+      name: 'sourceId',
```

Comment:
With regard to names with references, we've standardized on non 'id' in the name. 
So 
name: 'source' as the findSource(x) returns the object, not the id. 
Similar with targetId. 
as long as you don't have a lot of code dependent on this.

Reply:
ok sounds good, but these ones are generated by the relationship, should those be migrated to source/target?

Reply:
got it, ok, leave them.